js/index.js

```diff
@@ -39,4 +42,77 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+// Set Nav
+
+document.getElementsByTagName('nav')[0].getElementsByTagName('a')[0].innerText = siteContent["nav"]["nav-item-1"];
+document.getElementsByTagName('nav')[0].getElementsByTagName('a')[1].innerText = siteContent["nav"]["nav-item-2"];
+document.getElementsByTagName('nav')[0].getElementsByTagName('a')[2].innerText = siteContent["nav"]["nav-item-3"];
+document.getElementsByTagName('nav')[0].getElementsByTagName('a')[3].innerText = siteContent["nav"]["nav-item-4"];
+document.getElementsByTagName('nav')[0].getElementsByTagName('a')[4].innerText = siteContent["nav"]["nav-item-5"];
+document.getElementsByTagName('nav')[0].getElementsByTagName('a')[5].innerText = siteContent["nav"]["nav-item-6"];
+const nav = document.querySelector('nav');
```

Comment:
Way to use a query selector!

Reply:
You also could have looped through them and kept things DRYer, something like this

**
```const nav = document.querySelector('nav');
[...nav.children].forEach((current, index) => {
  current.innerHTML = siteContent['nav']['nav-item-' + (index + 1)];
});```

**